cg/meta/qc_metrics/collect_qc_metrics.py

```diff
@@ -83,11 +86,24 @@ def create_qc_metrics_request(self, case_id: str) -> CreateQCMetricsRequest:
             files=file_paths_and_tags,
         )
 
-    def get_qc_metrics(self, case_id) -> dict:
+    def get_case_qc_metrics(self, case_id) -> dict:
         """Get the qc metrics for a case."""
         qc_metrics_request: CreateQCMetricsRequest = self.create_qc_metrics_request(case_id)
         try:
             qc_metrics: dict = self.janus_api.qc_metrics(qc_metrics_request)
             return qc_metrics
         except (JanusClientError, JanusServerError) as error:
             LOG.info(f"Cannot collect qc metrics from Janus: {error}")
+
+    def get_create_case_request(self, case_id) -> CreateCaseRequest:
+        case_qc_metrics: dict = self.get_case_qc_metrics(case_id)
+        return CreateCaseRequest(**case_qc_metrics)
+
+    def create_case(self, case_id: str):
+        case_request: CreateCaseRequest = self.get_create_case_request(case_id)
+        try:
+            response: Response = self.arnold_api.create_case(case_request)
+            LOG.info(f"Created case for {case_id} in arnold. {response.status_code}")
+        except (ArnoldClientError, ArnoldServerError) as error:
+            LOG.info(f"Failed to create case in arnold: {error}.")
```

Comment:
Should be a warning or an error instead of info?

Reply:
Fixed to error